bundler/benchmarks/bm_bundle_install_rails.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'support/benchmark_bundler.rb'
+require 'bundler/cli'
+
+
+Dir.mktmpdir do |dir|
+  Dir.chdir(dir) do
+    File.write('Gemfile', <<-GEMFILE)
+      source 'https://rubygems.org'
+      gem 'rails', '~> 5.0.0'
+      gem 'nokogiri', '1.6.7.2'
+    GEMFILE
+
+    system("bundle install --quiet")
```

Comment:
You can completely remove the network from this by running `bundle pack` outside the benchmark, before running `bundle install`. In fact, we probably want one benchmark that installs a Gemfile of completely locked gems (for consistent comparison), and then a different benchmark that runs against a Gemfile without declared versions (to test the resolver, etc.).

At that point, I think the runtime will be dominated by the speed of unpacking the gems and copying the files—which is why it's good to have both `install` and `lock` in the benchmarks, so we can see changes/regressions in the resolver and the gem installation code.